cosi.proto

```diff
@@ -182,19 +182,15 @@ message ProvisionerCreateBucketRequest {
 }
 
 message ProvisionerCreateBucketResponse {
-    // Intentionally left blank
+    // BucketName returned here is expected to be the globally unique 
+	// identifier for the bucket in the object storage provider 
+	string BucketName = 1;
 }
 
 message ProvisionerDeleteBucketRequest {
     // This field is REQUIRED
     // Protocol specific information required by the call is passed in as key,value pairs.
     Protocol protocol = 1;
-
-    // This field is OPTIONAL
-    // Protocol specific information required by the call is passed in as key,value pairs.
-    // The caller should treat the values in parameters as opaque. 
-    // The receiver is responsible for parsing and validating the values.
-    map<string,string> parameters = 2;
```

Comment:
In ProvisionerDeleteBucketResponse (L200) should access_policy = 1 (not 4)?

Reply:
In ProvisionerGrantBucketAccessRequest (L204), I thought that protocol and parameters were being removed